src/main/java/com/google/sps/servlets/MealServlet.java

```diff
@@ -59,7 +59,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
             if (requestType.equals("similar")) {
                 // GET meal/similar
                 returnIdOfSimilar(request, response);
-            } else { 
+            } else if (requestType.contains("query=")) { 
+                // GET meal/query=
```

Comment:
I suppose `query` should be parameter:
```
// GET meal?query=<something>
```

Reply:
Agreed. If the request is `/meal?query=<something>`, then the pathInfo will be set to `"/meal"`, and you will be able to `request.getParameter("query")`, which should return `"<something>"`.

Reply:
Requests dont work when I use passing parameters, multiple calls of `doGet()` stopped only after I changed endpoint. 
For endpoint /meal function `searchMeal()` in the `body onload` fetches wrong data.